pandas/tests/extension/base/interface.py

```diff
@@ -21,7 +21,7 @@ def test_ndim(self, data):
         assert data.ndim == 1
 
     def test_can_hold_na_valid(self, data):
-        assert data._can_hold_na in {True, False}
+        assert data._can_hold_na  # Must be True
```

Comment:
you can ``assert data._can_hold_na is True``, otherwise this is not strictly checking that it is an actual boolean True